.circleci/config.yml

```diff
@@ -1511,12 +1511,22 @@ jobs:
           docker_layer_caching: true
       - run:
           name: Run Kontrol Tests
-          command: pnpm test:kontrol
+          command: |
+            curl -X POST \
+              -H "Accept: application/vnd.github+json" \
+              -H "Authorization: Bearer $RV_COMPUTE_TOKEN" \
+              https://api.github.com/repos/runtimeverification/optimism-ci/actions/workflows/optimism-ci.yaml/dispatches \
+              -d '{
+                "ref": "master",
+                "inputs": {
+                  "branch_name": "<<pipeline.git.branch>>",
+                  "extra_args": "script",
+                  "statuses_sha": "<< pipeline.git.revision >>",
+                  "org": "ethereum-optimism",
+                  "repository": "optimism"
+                }
+              }'
           working_directory: ./packages/contracts-bedrock
-      - store_artifacts:
-          path: ./packages/contracts-bedrock/test/kontrol/logs/kontrol-results_latest.tar.gz
-      - store_test_results:
-          path: ./packages/contracts-bedrock
```

Comment:
Are test results and artifacts are passed to CircleCI on completion, or do they need to be fetched from the github workflow job? This would be another good thing to document in the comments.

Additionally, we never got `store_test_results` working properly (to see individual test runtimes and insights) and I'm unsure why—will this PR resolve that?

Reply:
I'm happy to work with @raffaele-oplabs on it. However I don't have enough visibility into circleCI to debug it outright. 

Moving to this new CI flow, more than likely a post-op workflow is required to fetch the results to store them into CircleCI.